ports/nrf/boards/WRL13990/README.md

```diff
@@ -0,0 +1,72 @@
+# Nordic nRF5x micropython
```

Comment:
I think this page could be a good supplement for describing the board. For me it would made sense to skip this top level nRF5x title, keep the `WRL13990 Board Support` as title. The section below describes what chip is on the board.